### PR TITLE
new video URLs

### PR DESCRIPTION
**Pull request by ncawthon**

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
any chance we could make this commit "add new video urls for onsite docs" ? @ncawthon 

if no, i'm happy to change it after merge

**Comment by ncawthon** [comment]
https://github.com/npm/docs/commit/3a48e0965e6585ee3a1a9ddaab4783da8a329fb6

**Comment by ashleygwilliams** [comment]
i would just do `git rebase -i origin/master`
`delete 3a48e09`
`edit  3a48e09`

then `git commit --amend`
change the message to `add 2 new video urls for onsite and getting started docs`

then `git push origin new-vids -f`

**Comment by ashleygwilliams** [comment]
hi @ncawthon -- i rebased this for you. would you still like it to be merged? if so, i'm happy to do so- just lemme know.

File: content/enterprise/intro.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ with private modules.</p>
 
 <p>So let's set up your npm On-Site server:</p>
 
-<iframe width="640" height="480" src="//www.youtube.com/embed/XNr48kvz38o" frameborder="0" allowfullscreen></iframe>
+<iframe width="640" height="360" src="https://www.youtube.com/embed/mKMaG0cixXw" frameborder="0" allowfullscreen></iframe>
```
Comment (nexdrew) [inline content/enterprise/intro.md]: Height still renders as 480 locally, so not sure the height change is necessary.

Also, is there a particular reason to use https for the iframe here? Should all embedded videos follow suit?

**Comment by nexdrew** [comment]
Other than my [nit-picky comment](https://github.com/npm/docs/pull/467#discussion_r48486728), the video changes look good to me. :+1:

File: content/enterprise/intro.md (hunk 1 above)
Comment (ashleygwilliams) [inline content/enterprise/intro.md]: @nexdrew could you fix this height change and then rebase into the previous commit? my bet is it is copypasta

not sure about the https, but we are slowly changing everything to that, so maybe open an issue if the other vids are not https

Comment (nexdrew) [inline content/enterprise/intro.md]: Ok will do.

**Comment by ncawthon** [comment]
yes please @ashleygwilliams , thanks for helping my git floundering...

**Comment by nexdrew** [comment]
Ok, I think this PR now contains only the desired changes:

- `content/enterprise/intro.md`:
    - `//www.youtube.com/embed/XNr48kvz38o` -> `https://www.youtube.com/embed/mKMaG0cixXw`
- `content/getting-started/what-is-npm.md`
    - `//www.youtube.com/embed/pa4dc480Apo` -> `https://www.youtube.com/embed/x03fjb2VlGY`

Note that the "push" build failed b/c this branch is still using Node 0.10 and 0.11 in its Travis config. The "pr" build is the one that matters.

**Comment by nexdrew** [comment]
In the spirit of Always Be Closing, I'm gonna merge this.

**Pull request merged**